src/Compilers/Shared/BuildClient.cs

```diff
@@ -86,6 +89,37 @@ internal RunCompilationResult RunCompilation(IEnumerable<string> originalArgumen
             return new RunCompilationResult(exitCode);
         }
 
+        private static bool TryEnableMulticoreJitting(out string errorMessage)
```

Comment:
>out string errorMessage [](start = 54, length = 23)

Consider passing the TextWriter and writing to it instead.

Reply:
I'd prefer to centralize access to I/O as much as possible. I'm not even that pleased at directly accessing a TextWriter in this method, but due to the multitargeting and reference restrictions there isn't really a suitable abstraction I could put this in.